core/prisma/migrations/20241107193617_add_capabilities_to_db/migration.sql

```diff
@@ -0,0 +1,147 @@
+    ('admin'::"MemberRole", 'stage'::"MembershipType", 'runAction'::"Capabilities"),
+    ('admin'::"MemberRole", 'stage'::"MembershipType", 'viewStage'::"Capabilities"),
+    ('admin'::"MemberRole", 'stage'::"MembershipType", 'manageStage'::"Capabilities"),
+    ('admin'::"MemberRole", 'stage'::"MembershipType", 'addPubMember'::"Capabilities"),
+    ('admin'::"MemberRole", 'stage'::"MembershipType", 'removePubMember'::"Capabilities"),
+    ('admin'::"MemberRole", 'stage'::"MembershipType", 'addStageMember'::"Capabilities"),
+    ('admin'::"MemberRole", 'stage'::"MembershipType", 'removeStageMember'::"Capabilities"),
+    ('contributor'::"MemberRole", 'pub'::"MembershipType", 'viewPub'::"Capabilities"),
+    ('contributor'::"MemberRole", 'pub'::"MembershipType", 'updatePubValues'::"Capabilities"),
+    ('contributor'::"MemberRole", 'pub'::"MembershipType", 'editPubWithForm'::"Capabilities"),
+    ('editor'::"MemberRole", 'pub'::"MembershipType", 'movePub'::"Capabilities"),
+    ('editor'::"MemberRole", 'pub'::"MembershipType", 'viewPub'::"Capabilities"),
+    ('editor'::"MemberRole", 'pub'::"MembershipType", 'updatePubValues'::"Capabilities"),
+    ('editor'::"MemberRole", 'pub'::"MembershipType", 'createRelatedPub'::"Capabilities"),
+    ('editor'::"MemberRole", 'pub'::"MembershipType", 'editPubWithForm'::"Capabilities"),
+    ('admin'::"MemberRole", 'pub'::"MembershipType", 'movePub'::"Capabilities"),
+    ('admin'::"MemberRole", 'pub'::"MembershipType", 'viewPub'::"Capabilities"),
+    ('admin'::"MemberRole", 'pub'::"MembershipType", 'deletePub'::"Capabilities"),
+    ('admin'::"MemberRole", 'pub'::"MembershipType", 'updatePubValues'::"Capabilities"),
+    ('admin'::"MemberRole", 'pub'::"MembershipType", 'createRelatedPub'::"Capabilities"),
+    ('admin'::"MemberRole", 'pub'::"MembershipType", 'createPubWithForm'::"Capabilities"),
+    ('admin'::"MemberRole", 'pub'::"MembershipType", 'editPubWithForm'::"Capabilities"),
+    ('admin'::"MemberRole", 'pub'::"MembershipType", 'runAction'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'movePub'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'createPub'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'viewPub'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'deletePub'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'updatePubValues'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'createRelatedPub'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'createPubWithForm'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'editPubWithForm'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'runAction'::"Capabilities"),
+    ('editor'::"MemberRole", 'community'::"MembershipType", 'viewStage'::"Capabilities"),
```

Comment:
Probably no need to review below here, since community admins get every capability